cockroachcloud/quickstart.md

```diff
@@ -95,225 +99,134 @@ You will see a welcome message when you've successfully connected to your cluste
+    {{site.data.alerts.end}}
+
+1. Install the code dependencies:
+
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    node app.js
+    npm install
     ~~~
 
-    In the **Connection info** dialog, go to the **Connection string** tab, copy the connection string from step 2, and paste it in your terminal.
+1. Run the code:
 
-    ~~~
-    prompt: connectionString:
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    node app.js
     ~~~
 
-    After entering the connection string, the program will execute. The output will look like this:
+    The output will look like this:
 
     ~~~
     Hello world!
     ~~~
+
 </section>
 
 <section class="filter-content" markdown="1" data-scope="python">
 
-1. Install SQLAlchemy.
+1. Clone the `quickstart-code-samples` repo, and navigate to the `python` directory of the repo:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    pip install psycopg2-binary
+    git clone https://github.com/cockroachdb/quickstart-code-samples
     ~~~
 
-1. Create a `main.py` file and copy in the following code:
-
     {% include_cached copy-clipboard.html %}
-    ~~~ python
-    import logging
-    import os
-    import psycopg2
-
+    ~~~ shell
+    cd quickstart-code-samples/python
+    ~~~
 
-    def print_hello(conn):
-      with conn.cursor() as cur:
-          cur.execute("SELECT message FROM messages")
-          logging.debug("print_hello(): status message: %s", cur.statusmessage)
-          rows = cur.fetchall()
-          conn.commit()
-          for row in rows:
-              print(row[0])
+    The code sample in this directory connects to {{ site.data.products.db }} with the [psycopg2 driver](https://www.psycopg.org).
 
+1. Set the `DATABASE_URL` environment variable to the connection string that you [copied from the {{ site.data.products.cloud }} Console earlier](#step-2-connect-to-the-cluster):
 
-    def main():
-      conn_string = input('Enter a connection string:\n')
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    export DATABASE_URL="<connection-string>"
+    ~~~
 
-      conn = psycopg2.connect(os.path.expandvars(conn_string))
-      print_hello(conn)
+    The code sample uses the connection string stored in the environment variable `DATABASE_URL` to connect to your cluster.
 
-      # Close communication with the database.
-      conn.close()
+    {{site.data.alerts.callout_success}}
+    For reference information about connecting to CockroachDB with supported client drivers, see [Connect to a CockroachDB Cluster](../stable/connect-to-the-database.html?filters=python).
+    {{site.data.alerts.end}}
 
+1. Install psycopg2.
 
-    if __name__ == "__main__":
-        main()
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    pip3 install psycopg2-binary
```

Comment:
I would go with a `requirements.txt` for this. Also, on a newer system I think the pip for Python 3 will just be called `pip` rather than `pip3`.

Reply:
Done.